scripts/upload.sh

```diff
@@ -35,13 +35,7 @@ s3_upload()
     echo ""
     verbose_item "Uploading '$1'"
     # aws s3 cp "$DRY_RUN" --region us-west-1 "$1" "$REMOTE_PATH/$1"
```

Comment:
debugging? i guess... it's better to put an `echo` before the command to see the result. Will remove it.